src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -990,9 +990,11 @@ public void SetCookies(Uri uri, string cookieHeader)
         }
     }
 
+    // PathList needs to be public in order to maintain binary serialization compatibility as the System shim
+    // needs to have access to type-forward it.
     [Serializable]
     [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
-    internal struct PathList
+    public struct PathList
```

Comment:
This type isn't exposed in the reference assemblies which menas a) users won't see it b) it's not documented. So we are safe to make fundamental changes in the future as long as serialization compatibility is guaranteed.